src/core/varlink-common.c

```diff
@@ -66,7 +66,7 @@ int cpuset_build_json(sd_json_variant **ret, const char *name, void *userdata) {
 
         r = cpu_set_to_dbus(cpuset, &array, &allocated);
         if (r < 0)
-                return log_debug_errno(r, "Failed to call cpu_set_to_dbus(): %m");
+                return log_debug_errno(r, "Failed to serialize cpu set to dbus: %m");
```

Comment:
You're making this change in the wrong commit, that's where my confusion was coming from